dll/win32/hhctrl.ocx/help.c

```diff
@@ -1620,12 +1620,12 @@ static BOOL HH_CreateHelpWindow(HHInfo *info)
     wcex.cbClsExtra     = 0;
     wcex.cbWndExtra     = sizeof(LONG_PTR);
     wcex.hInstance      = hhctrl_hinstance;
-    wcex.hIcon          = LoadIconW(NULL, (LPCWSTR)IDI_APPLICATION);
+    wcex.hIcon          = LoadIconW(hhctrl_hinstance, MAKEINTRESOURCEW(IDI_HHICON));
     wcex.hCursor        = LoadCursorW(NULL, (LPCWSTR)IDC_ARROW);
     wcex.hbrBackground  = (HBRUSH)(COLOR_MENU + 1);
     wcex.lpszMenuName   = NULL;
     wcex.lpszClassName  = windowClassW;
-    wcex.hIconSm        = LoadIconW(NULL, (LPCWSTR)IDI_APPLICATION);
+    wcex.hIconSm        = NULL;
```

Comment:
Why don't you set the small icon?

Reply:
> If this member is NULL, the system searches the icon resource specified by the hIcon member for an icon of the appropriate size to use as the small icon.

[https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassexw](https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassexw)